buildpipeline/osx.groovy

```diff
@@ -0,0 +1,76 @@
+@Library('dotnet-ci') _
+
+// Incoming parameters.  Access with "params.<param name>".
+// Config - Build configuration. Note that we don't using 'Configuration' since it's used
+//          in the build scripts and this can cause problems.
+// OuterLoop - If true, runs outerloop, if false runs just innerloop
+
+def submittedHelixJson = null
+
+simpleNode('OSX10.12','latest') {
```

Comment:
@weshaggard OSX is still missing from pipelines.  I can add it in, but I also need to ensure that we have open queues for Helix for OSX.